modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -14,7 +14,7 @@ For distributed units (DUs), {rh-rhacm} supports {product-title} deployments tha
 
 * Install {product-title} {product-version} on a hub cluster.
 * Install {rh-rhacm} and create the `MultiClusterHub` resource.
-* Create persistent volume custom resources (CR) for database and file system storage.
+* Create persistent volume custom resources (CR) for database, file system, and image storage.
```

Comment:
Persistent volumes are not a CR. They're part of the base kubernetes API.

ref:
https://docs.openshift.com/container-platform/4.10/rest_api/workloads_apis/persistentvolume-v1.html#persistentvolume-v1
https://kubernetes.io/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1/